MVCapplication/src/main/java/com/softserve/if072/mvcapp/service/GoShoppingPageService.java

```diff
@@ -25,13 +25,19 @@
 public class GoShoppingPageService {
     private static final Logger LOGGER = LogManager.getLogger(GoShoppingPageService.class);
     private final RestTemplate restTemplate;
+    private final CartService cartService;
+    private final MessageService messageService;
+    private static final String LOCALE_COOKIE = "myLocaleCookie";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "LOCALE_COOKIE" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)